test/Microsoft.AspNet.Mvc.Core.Test/VirtualFileResultTest.cs

```diff
@@ -241,73 +267,29 @@ public void Constructor_SetsContentTypeAndParameters()
             Assert.Equal(expectedData, contents);
         }
 
-        [Theory]
```

Comment:
Not quiet parsing your comment. Shouldn't we still at least test a few longer paths, including those that start with dots and slashes?